packages/lib/survey/service.ts

```diff
@@ -439,8 +509,10 @@ export async function createSurvey(environmentId: string, surveyBody: TSurveyInp
 
   captureTelemetry("survey created");
 
-  revalidateTag(getSurveysCacheTag(environmentId));
-  revalidateTag(getSurveyCacheTag(survey.id));
+  surveyCache.revalidate({
```

Comment:
`TSurveyInput` could also have triggers and attributeFilters so we also need to revalidate those :-)